src/serialization_utils.c

```diff
@@ -142,6 +142,7 @@ int encodeString(char *p, size_t sz, RedisModuleString *str)
 
     if (str == NULL) {
         len = 0;
+        e = "";
```

Comment:
Just a question, can we avoid these null checks altogether that we've introduced for ACL? Can we use empty string for non-existing ACLs rather than NULL? Does it make things worse?

Reply:
its not a "char *" its a RedisModuleString so would involve an allocation and free every single time, I think a NULL check is better